src/core/scrollcomponent/BaseScrollComponent.tsx

```diff
@@ -25,4 +25,9 @@ export default abstract class BaseScrollComponent extends React.Component<Scroll
     public getScrollableNode(): number | null {
         return null;
     }
+
+    //Override and return ref to your custom scrollview. Useful if you need to use Animated Events on the new architecture.
+    public getNativeScrollRef(): unknown | null {
```

Comment:
Does it need to be unknown?

Reply:
I don't really remember why I made it this way. Updated to match the type of `FlatList`.